test-support/helpers/ember-prop-types/index.js

```diff
@@ -0,0 +1,153 @@
+import {expect} from 'chai'
+import Ember from 'ember'
+import {beforeEach, it} from 'mocha'
+
+const {
+  isEmpty,
+  typeOf
+} = Ember
+
+/**
+ * Create error messages for each failure reason
+ *
+ * @param {Array} different propTypes that are specified but not correctly
+ * @param {Array} missing propTypes that are expected but are missing
+ * @param {Array} unexpected propTypes that are not expected
+ * @returns {Array} failure reasons
+ */
+function createMessages (different, missing, unexpected) {
+  const messages = []
+
+  if (!isEmpty(different)) {
+    messages.push(
+      `The following propTypes are different than expected: ${different.join(', ')}`
+    )
+  }
+
+  if (!isEmpty(missing)) {
+    messages.push(
+      `The following propTypes were not found but are expected: ${missing.join(', ')}`
+    )
+  }
+
+  if (!isEmpty(unexpected)) {
+    messages.push(
+      `The following propTypes were found but not expected in the test: ${unexpected.join(', ')}`
```

Comment:
Why does this message include the text "in the test" while the other two do not? It seems like we should be consistent. I've vote for not including the text "in the test" anywhere, but if we're gonna have it, it should probably be in all three messages.